MosipNexus/crawler/jira_crawler.py

```
@@ -0,0 +1,153 @@
+        body = (c.get("body") or "").strip()
+        author = c.get("author", {}).get("displayName", "")
+        if body:
+            comment_bodies.append(f"[{author}]: {body}")
+
+    content_parts = []
+    if desc:
+        content_parts.append(f"Problem:\n{desc}")
+    if resolution_name:
+        content_parts.append(f"Resolution: {resolution_name}")
+    if comment_bodies:
+        content_parts.append(f"Discussion:\n" + "\n\n".join(comment_bodies))
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)